low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -75,6 +91,14 @@ def get_health_info(self, rpath):
         fru_found = False
         nodes = rpath.strip().split(">")
         leaf_node, _ = self.get_node_details(nodes[-1])
+        try:
+            info["name"] = Conf.get("cortx_build_info", "NAME")
+            info["version"] = Conf.get("cortx_build_info", "VERSION")
+            info["build"] = Conf.get("cortx_build_info", "BUILD")
+            info["os"] = Conf.get("cortx_build_info", "OS")
+        except ConfError as e:
+            # TODO add logs
+            print(f"something went wrong: {e}")
```

Comment:
Unable to get OS & CORTX Build info

Reply:
Updated, although it's just a place holder to avoid Codacy warning. Once logs are in place, I will remove this print statement.

Reply:
oh right, comment out print